client/ayon_maya/plugins/publish/extract_pointcache.py

```diff
@@ -260,6 +267,16 @@ def process(self, instance):
     def get_members_and_roots(self, instance):
         return instance[:], instance.data.get("setMembers")
 
+    def get_members_shapes(self, instance):
+        members = instance.data("setMembers")
+        members = cmds.ls(members,
+                          dag=True,
+                          shapes=True,
+                          type=("mesh", "nurbsCurve"),
+                          noIntermediate=True,
+                          long=True)
+        return members
```

Comment:
FYI, This logic may fail for instanced geometry. Also shouldn't the members already have been collected by the collector, essentially making this:

```
nodes = list(instance[:])
return cmds.ls(nodes, type=("mesh", "nurbsCurve"), long=True)
```
Right?